docs/pages/developers/network/collator.mdx

```diff
@@ -0,0 +1,145 @@
+
+**Action 2: Generate session Keys**: On your running collator node, you will need to generate your session keys by making an `author_rotateKeys` RPC Call.
+You can do this with a curl command:
+
+
+```bash
+curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "author_rotateKeys", "params":[]}' http://localhost:9990
+```
+
+Note: Replace 9990 with your node's RPC port if it is different.
+This command will return a long hex string. This is your session key public key
+
+**Action 3: Link your keys**: You must now tell the blockchain that your main  account (`AccountId`) will use these new session keys for collating.
+You do this by calling the `session.setKeys` extrinsic.
+
+- Navigate to `Developer -> Extrinsics` in the Polkadot UI.
+- Select your main account(the one with the reputation score).
+    Note that this is your relayer's public key. It must be the same account that is configured as the `signer` in your relayer setup and is the account that has been accruing reputation.
+- Choose the `session` pallet and the `setKeys` extrinsic.
+- Paste the entire hex string from the previous step into the `keys` field.
+- Leave the `proof` field empty (0x00).
+- Sign and submit the transaction.
+
+<figure>
+  <img src="/set_collator_keys.png" alt="Setting session keys" />
+  <figcaption style={{ textAlign: "center" }}>How to set Session Keys</figcaption>
+</figure>
+
+
+### Step 3: Register as a Candidate
+Now that the node is ready and your keys are linked, you can now declare your intention to becoming a Collator.
+
+**Action**: Call the `collatorSelection.registerAsCandidate` extrinsic.
+
+- Navigate to `Developer -> Extrinsics` in the Polkadot-JS UI.
+- Select your main account.
+     Note that this is your relayer's public key. It must be the same account that is configured as the `signer` in your relayer setup and is the account that has been accruing reputation.
+- Choose the `collatorSelection` pallet and the `registerAsCandidate` extrinsic.
+- Sign and submit the transaction.
+
+<figure>
+  <img src="/register_as_candidate.png" alt="Registering as candidate" />
+  <figcaption style={{ textAlign: "center" }}>How to register as candidate</figcaption>
+</figure>
+
+
+This transaction will enable you to enter the candidacy pool.
+
+## The selection process & what happens next
+
+At the beginning of each new session, our `pallet-collator-manager` runs it's selection logic:
+
+- It fetches the list of all registered candidates who have set their sesssion keys.
+- It ranks them from highest to lowest based on their Reputation Asset balance.
+- It selects the top candidates to fill the available collator slots for the new session.
+
+If you are selected:
+
+- Your Reputation Asset balance will be burned. This is somewhat like your stake for the session.
+- Your node will begin authoring and producing blocks after 2 rounds(sessions). To do this, it must be running with the `--validator` flag
```

Comment:
I think you should tell them to restart ther node with the validator flag after the entire setup is completed.